sdk/src/priorityFee/priorityFeeSubscriber.ts

```diff
@@ -0,0 +1,62 @@
+import { Connection, PublicKey } from '@solana/web3.js';
+
+export class PriorityFeeSubscriber {
+	connection: Connection;
+	frequency: number;
+	addresses: PublicKey[];
+
+	intervalId?: NodeJS.Timer;
+
+	avg = 0;
+	max = 0;
+
+	public constructor({
+		connection,
+		frequency,
+		addresses,
+	}: {
+		connection: Connection;
+		frequency: number;
+		addresses: PublicKey[];
+	}) {
+		this.connection = connection;
+		this.frequency = frequency;
+		this.addresses = addresses;
+	}
+
+	public async subscribe(): Promise<void> {
+		if (this.intervalId) {
+			return;
+		}
+
+		this.intervalId = setInterval(this.load.bind(this), this.frequency);
+	}
+
+	public async load(): Promise<void> {
+		// @ts-ignore
+		const rpcJSONResponse: any = await this.connection._rpcRequest(
+			'getRecentPrioritizationFees',
+			[this.addresses]
+		);
+
+		let sum = 0;
+		let max = 0;
+
+		for (const { prioritizationFee } of rpcJSONResponse.result) {
+			sum += prioritizationFee;
+			max = Math.max(max, prioritizationFee);
```

Comment:
i'd shy away using the max `prioritizationFee` of the response since that can create a sticky effect on overall gas prices if drift SDK users become a large chunk of block space.

I think something like the `prioritizationFee` of the latest slot will be sufficient (i..e if last block had a priority fee of 1000, then this class should return 1000).

Also note that the response from `getRecentPrioritizationFees` is not sorted, so you will need to sort by `slot` to get the above